apps/project/schema.py

```diff
@@ -121,6 +126,38 @@ def get_top_entity_contributor(project, Entity):
     ]
 
 
+def get_project_stats_summary(self):
+    projects = Project.get_for_member(self.context.request.user)
+    # Lead stats
+    leads = Lead.objects.filter(project__in=projects)
+    total_leads_tagged_count = leads.annotate(entries_count=models.Count('entry')).filter(entries_count__gt=0).count()
+    total_leads_tagged_and_controlled_count = leads.annotate(
+        controlled_entries_count=models.Count(
+            'entry', filter=models.Q(entry__controlled=True)
+        )
+    ).filter(controlled_entries_count__gt=0).count()
+    # Entries activity
+    recent_projects_id = list(
+        projects.annotate(
+            entries_count=Cast(KeyTextTransform('entries_activity', 'stats_cache'), models.IntegerField())
+        ).filter(entries_count__gt=0).order_by('-entries_count').values_list('id', flat=True)[:3])
+    recent_entries = Entry.objects.filter(
+        project__in=recent_projects_id,
+        created_at__gte=(timezone.now() + relativedelta(months=-3))
+    )
+    recent_entries_activity = recent_entries.order_by('created_at__date').values('created_at__date').annotate(
+        count=models.Count('*')
+    ).values('project_id', 'count', date=models.Func(models.F('created_at__date'), function='DATE'))
```

Comment:
Easy read
```suggestion
    recent_projects_id = list(
        projects.annotate(
            entries_count=Cast(
                KeyTextTransform('entries_activity', 'stats_cache'),
                models.IntegerField()
            ),
        ).filter(entries_count__gt=0)
        .order_by('-entries_count')
        .values_list('id', flat=True)[:3]
    )

    recent_entries = Entry.objects.filter(
        project__in=recent_projects_id,
        created_at__gte=(timezone.now() + relativedelta(months=-3))
    )

    recent_entries_activity = (
        recent_entries
        .order_by('created_at__date').values('created_at__date').annotate(
            count=models.Count('*')
        ).values(
            'project_id',
            'count',
            date=models.Func(models.F('created_at__date'), function='DATE'),
        )
    )
```

Reply:
![image](https://github.com/the-deep/server/assets/7059255/eb358e85-d2f2-4630-9aa4-dc838f80caa6)